app/assets/stylesheets/components/_card.scss

```diff
@@ -82,16 +77,10 @@ color: white;
   text-shadow: 1px 1px 3px rgba(0,0,0,0.2);
   border-radius: 5px;
   box-shadow: 0 0 15px rgba(0,0,0,0.2);
-  position: absolute;
-  top: 5px;
-  z-index: 0;
-}
 
-.pet-card {
-  background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.3));
 }
 
-.main-container {
-  margin-top: 24px;
-  display: flex;
+.card-text a {
```

Comment:
We are using scss that allows us to nest styling. Can you refactor ".card-text a" to nest it into ".card-text"?